packages/site/src/pages/index.tsx

```diff
@@ -324,35 +371,38 @@ const Index = () => {
     let transactionDetails: Record<string, any> = {
       payload: {
         to: targetAccount,
-        value: transferAmount,
+        value: parseUnits(transferAmount, 'ether'), // as it's ethers
         data: '0x'
       },
-      account: snapState.accounts[0]?.id || '', // TODO: need to use currently selected snap account
+      account: selectedAccount?.id,
```

Comment:
just checking - do you need `selectedAccount?.id || '',` in all of these?

Reply:
yes it's looking for id not address!